src/ensembl/src/content/app/entity-viewer/gene-view/components/protein-domain-image/ProteinDomainImage.tsx

```diff
@@ -32,6 +32,7 @@ const TRACK_HEIGHT = 24;
 
 export type ProteinDomainImageProps = {
   transcriptId: string;
+  refCDSLength: number;
```

Comment:
let's remove the `ref` part of the name so that it doesn't imply that you are passing a React ref. `cdsLength` should be sufficient

Reply:
`cdsLength` is not accurate as it is not the length of the protein's CDS. We need to an alternative name for this.